deploy/examples/images.txt

```diff
@@ -3,6 +3,8 @@
  k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
  k8s.gcr.io/sig-storage/csi-resizer:v1.4.0
  k8s.gcr.io/sig-storage/csi-snapshotter:v5.0.1
+ k8s.gcr.io/sig-storage/livenessprobe:v2.5.0
+ mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0
```

Comment:
dont we have this in `k8s.gcr.io`?

Reply:
> dont we have this in `k8s.gcr.io`?

Official nfs example uses mcr.microsoft.com registry https://github.com/kubernetes-csi/csi-driver-nfs/blob/1177e9be8a0e70648d1bcc9db6b82911317d90a0/deploy/v3.1.0/csi-nfs-node.yaml#L83 

It is not available at that registry

```
[rakshith@fedora ceph-csi]$ docker pull k8s.gcr.io/sig-storage/nfs-csi:v3.1.0
Error response from daemon: manifest for k8s.gcr.io/sig-storage/nfs-csi:v3.1.0 not found: manifest unknown: Failed to fetch "v3.1.0" from request "/v2/sig-storage/nfs-csi/manifests/v3.1.0".
[rakshith@fedora ceph-csi]$ docker pull k8s.gcr.io/sig-storage/nfs-csi
Using default tag: latest
Error response from daemon: manifest for k8s.gcr.io/sig-storage/nfs-csi:latest not found: manifest unknown: Failed to fetch "latest" from request "/v2/sig-storage/nfs-csi/manifests/latest".
```